doc/sphinx/source/theory/theoryparamsinfo.rst

```diff
@@ -0,0 +1,95 @@
+  PTO                                                        2
+  FNS                                                  FONLL-C
+  DAMP                                                       0
+  IC                                                         1
+  ModEv                                                    TRN
+  XIR                                                        1
+  XIF                                                        1
+  NfFF                                                       5
+  MaxNfAs                                                    5
+  MaxNfPdf                                                   5
+  Q0                                                      1.65
+  alphas                                                 0.118
+  Qref                                                    91.2
+  QED                                                        0
+  alphaqed                                          0.00749625
+  Qedref                                                 1.777
+  SxRes                                                      0
+  SxOrd                                                     LL
+  HQ                                                      POLE
+  mc                                                      1.51
+  Qmc                                                     1.51
+  kcThr                                                      1
+  mb                                                      4.92
+  Qmb                                                     4.92
+  kbThr                                                      1
+  mt                                                     172.5
+  Qmt                                                    172.5
+  ktThr                                                      1
+  CKM        0.97428 0.22530 0.003470 0.22520 0.97345 0.041...
+  MZ                                                   91.1876
+  MW                                                    80.398
+  GF                                               1.16638e-05
+  SIN2TW                                               0.23126
+  TMC                                                        1
+  MP                                                     0.938
+  Comments                               NNPDF3.1 NNLO central
+  global_nx                                                  0
+  EScaleVar                                                  1
+
+Some of the entries of the table have been truncated to fit in the terminal, for
+example the CKM matrix elements. If you want to see the
+full output or wish to keep a permanent copy of the table then you can use the
+command line option ``-d``, which stands for ``--dumptable`` which will save the
+table in your current working directory as ``theory_<id>_info.csv``::
+
+  $vp-checktheory --fit FIT
+
+where ``FIT`` is a valid fit name. If the fit cannot be found locally, the script
+will attempt to download it.
```

Comment:
I am not sure I understand this part. Do you want to highlight the use of `-d` or checking the theory settings for a given fit or both? For the last two, a sentence introducing it would be useful.

Reply:
I will not update these docs...  I just don't want to keep them too outdated but (specially taking into account the theory might change) I'd rather not touch them too much.